test/controllers/tmf-apis/catalog.js

```diff
@@ -2909,9 +2922,100 @@ describe('Catalog API', function() {
             parentCategoryRequest, { name: categoryName, parentId: parentId, isRoot: false }, 500, errorMsg, done);
     });
 
+    fdescribe('Index in pre validation', function() {
+        var myHelper = function myHelper(done, sName, results, url, pathname, query, expectedUrl, expectedQuery) {
```

Comment:
This method is actually executing the test, it sould be called something like testIndexMiddleware